addons/token-exchange/manifests/spoke_role.yaml

```diff
@@ -19,3 +19,6 @@ rules:
 - apiGroups: ["coordination.k8s.io"]
   resources: ["leases"]
   verbs: ["get", "list", "watch", "create", "update", "delete"]
+- apiGroups: ["ocs.openshift.io"]
+  resources: ["storageclusters"]
+  verbs: ["get", "list", "patch", "update"]
```

Comment:
We need to read storageclusters from all namespace, so it should be a clusterrole.
PR 33 already has enough access. Was it still not working?

Reply:
Could be just wrong apiGroup name (`storageclusters.ocs.openshift.io` ) in the PR 33.

Reply:
ok yes. Fix it in that PR.

Reply:
closing this PR, it is already taken care in PR 33